### PR TITLE
additional handling in nb convertor 

### PR DESCRIPTION
**Pull request by fishera123**
updated: 
- added handling of tags
- added handling of the section with different levels.  
- added handling for TeX type

### CODE REVIEW

File: kernel/base/src/main/java/com/twosigma/beaker/jupyter/commands/MagicCommand.java [hunk 1]
```diff
@@ -63,6 +64,10 @@ private void buildCommands() {
       String result = executeBashCode(code.replace("%%bash", ""));
       messageCreator.createMagicMessage(messageCreator.buildOutputMessage(message, result, false), executionCount, message);
     });
+    commands.put("%%latex", (code, message, executionCount) -> {
```
Comment (scottdraves) [inline kernel/base/src/main/java/com/twosigma/beaker/jupyter/commands/MagicCommand.java]: don't add this, instead generate a markdown cell that contains the math.

File: beaker/bkr2ipynb.py [hunk 2]
```diff
@@ -29,12 +29,24 @@
...
+
+
 def convertNotebook(notebook):
+  available_kernels = ['IPython', 'Groovy', 'Java', 'Scala']
   nb = new_notebook()
   with open(notebook) as data_file:
     data = json.load(data_file)
   evaluators = list((cell['evaluator']) for cell in data['cells'] if 'evaluator' in cell)
   kernel_name = max(evaluators, key=evaluators.count)
+  if kernel_name not in available_kernels:
```
Comment (scottdraves) [inline beaker/bkr2ipynb.py]: don't do this

**Comment by scottdraves** [comment]
handle tags like this: https://github.com/twosigma/beakerx/issues/5106

**Comment by fishera123** [comment]
removed %%latex magic and added tags handling

**Comment by scottdraves** [comment]
thx -- python style is 4 spaces for indentation.

**Pull request merged**